src/config/pci-passthrough.md

```diff
@@ -0,0 +1,405 @@
+GPU and it's audio device by running the following command:
+
+```
+$ lspci -nnk -d 1002:67df 
+07:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [1002:67df] (rev e7) 
+        Subsystem: Sapphire Technology Limited Nitro+ Radeon RX 570/580/590 [1da2:e366] 
+        Kernel driver in use: vfio-pci 
+```
+
+```
+$ lspci -nnk -d 1002:aaf0 
+07:00.1 Audio device [0403]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere HDMI Audio [Radeon RX 470/480 / 570/580/590] [1002:aaf0] 
+        Subsystem: Sapphire Technology Limited Device [1da2:aaf0] 
+        Kernel driver in use: vfio-pci 
+```
+
+The device ids we specified after the `vfio-pci.ids=` parameter should have the
+`vfio-pci` kernel driver in use.
+
+## Setting up OVMF-based guest VM
+
+Install `qemu`, `libvirt`, and `virt-manager` from void's official repositories.
+You must compile the `ovmf` package from source using xbps-src (see [building
+instructions](../packages/building.md)) after applying the patch located here:
+
+[New package: ovmf](https://github.com/void-linux/void-packages/pull/17225).
+
+Enable the following services with runit: `libvirtd`, `virtlogd`, and
+`virtlockd`.
+
+To run `virt-manager` without root, add yourself to the `libvirt` user group and
+uncomment the following lines in libvirt config `/etc/libvirt/libvirt.conf`:
+
+```
+unix_sock_group = "libvirt"
+unix_sock_ro_perms = "0770"
+unix_sock_rw_perms = "0770"
+unix_sock_admin_perms = "0770"
+unix_sock_dir = "/var/run/libvirt"
+```
+
+Logout and log back in to ensure the new group settings are setup.
+
+Open `virt-manager` and create a new virtual machine. Go through all the steps
+as normal except at the end: check the "Customize before install" checkbox.
+
+- In "Overview" section set firmware to UEFI
+- In overview set chipset to Q35. See: ([Tips for PCI Passthrough -
+   momi.ca](https://momi.ca/2019/11/17/pcipassthrough.html#org44297d6))
+- In "CPUs" section, under Configuration, set "Copy host CPU configuration".
+
+Then finish your installation.
+
+Now that the install is done, add your PCI devices to your virtual machine. Open
+your virtual machine and click "Add Hardware" and switch to the "PCI Host
+Device". Add the devices we specified in the `vfio-pci.ids=` kernel parameter.
+
+Additionally, you may want to add a mouse and keyboard. You can do this by
+clicking "Add Hardware" and going to the USB Host Device section.
+
+> Note: To avoid having to use two mice and keyboard, I use a KVM switch and
+> passthrough one of my USB controllers. Your IOMMU groups may or may not allow
+> you to do this.
+
+Boot up your new virtual machine with your install medium and install your
+operating system.
+
+> Note: To install your guest operating system, make sure you plug a monitor
+> into the GPU you passed through.
+
+## Additional Tweaks
+
+> Note: You can edit your virt-manager config by running: `sudo virsh edit
```

Comment:
As per the style guide, commands that need to be run as root should not use `sudo`:

```
# virsh edit <your-vm-name`
```